scripts/setup.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
is his all the same as from notebook_entrypoint other than the hive stuff at the end?

Reply:
yes. exactly copied over except for the last few line regarding hive-site.xml

Reply:
:+1: